Dockerfile

```diff
@@ -20,7 +20,7 @@ RUN apt-get -qq update \
     twine \
     jq \
     unzip \
-    openjdk-11-jdk \
+    openjdk-17-jdk \
     maven \
     elixir \
   && curl -fsSL https://packages.microsoft.com/config/debian/10/packages-microsoft-prod.deb -o /tmp/packages-microsoft-prod.deb \
```

Comment:
this probably needs updating as well as the references to `buster` below

Reply:
this page only shows 11 as supported
https://learn.microsoft.com/en-us/dotnet/core/install/linux-debian
but this exists
https://packages.microsoft.com/config/debian/12/
so I guess their docs are outdated.

Either way this also works so it's not really necessary but I can bump.